libs/platform/src/lib/components/form/radio-group/radio-group.component.ts

```diff
@@ -40,8 +43,20 @@ let nextUniqueId = 0;
     providers: [{ provide: FormFieldControl, useExisting: RadioGroupComponent, multi: true }]
 })
 export class RadioGroupComponent extends CollectionBaseInput implements AfterViewInit, AfterContentChecked, OnDestroy {
-    /** Value of selected radio button */
+    /**
+     * RBG label for accessibility
+     */
     @Input()
+    ariaLabel: string;
```

Comment:
Let's move this up as we already commented in the other PR2989

Reply:
Done